docker/Dockerfile

```diff
@@ -0,0 +1,29 @@
+ARG PHP_VERSION=7.2
+FROM php:${PHP_VERSION}-alpine
+
+ARG XDEBUG=0
```

Comment:
Just to note: this is really important !! MUST BE HERE, cannot be above `FROM` :) Thanks @TomHAnderson !!